akka-stream/src/main/scala/akka/stream/StreamRefs.scala

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright (C) 2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream
+
+import akka.NotUsed
+import akka.actor.{ ActorRef, ActorSystem }
+import akka.stream.scaladsl.{ Sink, Source }
+
+import scala.language.implicitConversions
+
+/**
+ * See full documentation on [[SinkRef]].
```

Comment:
Will that link actually help? Or does it just link to the object again?

Reply:
I believe this links to the class